### PR TITLE
Update GH URL with filter to all labels

### PR DESCRIPTION
**Pull request by vfrsilva**

### CODE REVIEW

File: docs/test-world-2/bug-reporting.mdx [hunk 1]
```diff
@@ -18,12 +18,12 @@ The Program gathers experienced node operators to provide the network backbone f
 
 ## Reporting Bugs
 
-All participants are  expected to raise any abnormal behavior during the protocol performance testing on [Github](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+OR%3A%3A%3A%3A%3A+label%3ATestworld-2-0-block-producer%2CTestworld-2-0-archive-node%2CTestworld-2-0-load-tester%2CTestworld-2-0-protocol-performance-testing%2CTestworld-2-0-snark-worker).
+All participants are  expected to raise any abnormal behavior during the protocol performance testing on [Github](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+).
```
Comment (barriebyron) [inline docs/test-world-2/bug-reporting.mdx:21]:
```suggestion
All participants are expected to raise any abnormal behavior during the protocol performance testing on the Mina [GitHub](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+) repository.
```

File: docs/test-world-2/bug-reporting.mdx [hunk 2]
```diff
@@ -18,12 +18,12 @@ The Program gathers experienced node operators to provide the network backbone f
...
+- The Block Producer is expected to raise any abnormal behavior during the protocol performance testing on [Github](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+), using the labels **Testworld-2-0-protocol-performance-testing** and **Testworld-2-0-block-producer**
+- The Load Tester is expected to raise any abnormal behavior during the protocol performance testing on [Github](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+), using the labels **Testworld-2-0-protocol-performance-testing** and **Testworld-2-0-load-tester**
```
Comment (barriebyron) [inline docs/test-world-2/bug-reporting.mdx:24]:
```suggestion
- **Block Producers** are expected to raise any abnormal behavior during the protocol performance testing on the Mina Protocol [GitHub](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+) repository, using both of these labels:

    - **Testworld-2-0-protocol-performance-testing** 
    - **Testworld-2-0-block-producer**

- **Load Testers** are expected to raise any abnormal behavior during the protocol performance testing on Mina Protocol [GitHub](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+) repository, using both of these labels:

    - **Testworld-2-0-protocol-performance-testing**
    - **Testworld-2-0-load-tester**
```

File: docs/test-world-2/bug-reporting.mdx [hunk 3]
```diff
@@ -18,12 +18,12 @@ The Program gathers experienced node operators to provide the network backbone f
...
+- The SNARK worker Operator is expected to raise any abnormal behavior during the protocol performance testing on [Github](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+), using the labels **Testworld-2-0-protocol-performance-testing** and **Testworld-2-0-snark-worker**
```
Comment (barriebyron) [inline docs/test-world-2/bug-reporting.mdx:25]:
```suggestion
- **SNARK Worker Operators** are expected to raise any abnormal behavior during the protocol performance testing on Mina Protocol [GitHub](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+) repository, using both of these labels:

    - **Testworld-2-0-protocol-performance-testing**
    - **Testworld-2-0-snark-worker**
```

File: docs/test-world-2/bug-reporting.mdx [hunk 4]
```diff
@@ -18,12 +18,12 @@ The Program gathers experienced node operators to provide the network backbone f
...
+- The Archive Node Operator is expected to raise any abnormal behavior during the protocol performance testing on [Github](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+), using the labels **Testworld-2-0-protocol-performance-testing** and **Testworld-2-0-archive-node**
```
Comment (barriebyron) [inline docs/test-world-2/bug-reporting.mdx:26]:
```suggestion
- **Archive Node Operators** are expected to raise any abnormal behavior during the protocol performance testing on the Mina Protocol [GitHub](https://github.com/MinaProtocol/mina/issues?q=is%3Aopen+label%3ATestworld-2-0-protocol-performance-testing%2CTestworld-2-0-block-producer%2CTestworld-2-0-load-tester%2CTestworld-2-0-snark-worker%2CTestworld-2-0-archive-node+), using both of these labels:

- **Testworld-2-0-protocol-performance-testing**
- **Testworld-2-0-archive-node**
```

**Review by barriebyron — APPROVED** [review]
LGTM, I made some structure updates so readers can see the labels more clearly

**Review by vfrsilva — COMMENTED** [review]
LGTM thank you

**Pull request merged**